src/hotspot/share/utilities/quickSort.hpp

```diff
@@ -26,21 +26,19 @@
 #define SHARE_UTILITIES_QUICKSORT_HPP
 
 #include "memory/allStatic.hpp"
-#include "runtime/globals.hpp"
 #include "utilities/debug.hpp"
+#include "utilities/globalDefinitions.hpp"
 
 class QuickSort : AllStatic {
 
  private:
   template<class T>
-  static void swap(T* array, size_t x, size_t y) {
-    T tmp = array[x];
-    array[x] = array[y];
-    array[y] = tmp;
+  static void swap_elements(T* array, size_t x, size_t y) {
+    swap(array[x], array[y]);
   }
 
   // As pivot we use the median of the first, last and middle elements.
-  // We swap in these three values at the right place in the array. This
+  // We swap these three values into the right place in the array. This
   // means that this method not only returns the index of the pivot
   // element. It also alters the array so that:
```

Comment:
Pre-existing nit: this should be one sentence: "... element, it also ..."

Reply:
I ended up doing a rewrite of the description.